featuretools/entityset/serialize.py

```diff
@@ -19,7 +19,7 @@
     import tempfile
 
 FORMATS = ['csv', 'pickle', 'parquet']
-SCHEMA_VERSION = "1.0.0"
+SCHEMA_VERSION = "1.1.0"
```

Comment:
@rwedge Do we need to bump the ES schema or feature_base schema. Or Both?

Reply:
I think only feature_base schema.